components/Chain/ScorecardListItem.tsx

```diff
@@ -6,15 +6,15 @@ import { MasteryIcons } from "../../styles/MasteryIcons";
 import { AntDesign } from "@expo/vector-icons";
 
 type Props = {
-	id: string;
-	title: string;
-	score: number;
+	attemps: {}[];
```

Comment:
attemps?  Is that a spelling mistake?

Reply:
Looks like a bizarre data structure {}[]

Reply:
@danfunk Yes, misspelling.

It's Typescript's indication of an array of objects.